test/test_pipeline/test_tabular_regression.py

```diff
@@ -300,6 +299,7 @@ def test_pipeline_score(fit_dictionary_tabular_dummy):
 
     pipeline = TabularRegressionPipeline(
         dataset_properties=fit_dictionary_tabular_dummy['dataset_properties'],
+        random_state=1
```

Comment:
Why integer?

Reply:
because we convert a seed with a random state instance with [this](https://github.com/automl/Auto-PyTorch/blob/b0e444f813417377a8c0721c573edc6a3f7ff307/autoPyTorch/pipeline/base_pipeline.py#L76) line.